### PR TITLE
clean up ui (icons, forms, colors/states)

### PR DESCRIPTION
**Pull request by lvuch**
-remove icons from side nav
image

-form controls: hover, focus, disabled, filled, default, error

image
image

-growl

### CODE REVIEW

File: report.20210826.122933.79412.0.001.json [hunk 1]
```diff
@@ -0,0 +1,797 @@
+
```
Comment (nwmac) [inline report.20210826.122933.79412.0.001.json:1]: This file should not here - this is another report. file as well that should go

File: .vscode/settings.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   "eslint.packageManager": "yarn",
   "eslint.validate": ["vue","html","javascript","typescript"],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": false
```
Comment (nwmac) [inline .vscode/settings.json:24]: Should this be changed? This will change for everyone.

**Review by nwmac — APPROVED** [review]
@lvuch I think given the churn in styling, we should try and do one PR per styling change - I'd rather see more small PRs that fix specific issues - also easier to unpick if we decide we need to revert something.

Can you add a description or screen of what the style change is as well

**Comment by nwmac** [comment]
@lvuch Can you split this PR up - the title doesn't really match what is in the PR.

I would suggest 3 PRs - 

1. Update presentation of side navigation - currently this is just removing the 2nd-level icons, but before we merge, I think we need to add icons for the top-level
2. Update LabeledInput control - I think we need to look at this in conjunction with other form controls
3. Improve presentation of Growl control

I think for all, we should add an example to the Storybook and include a screen shot in the PR

**Comment by lvuch** [comment]
Update presentation of side navigation - currently this is just removing the 2nd-level icons, but before we merge, I think we need to add icons for the top-level
https://github.com/rancher/dashboard/pull/5162

**Comment by lvuch** [comment]
Update LabeledInput control - I think we need to look at this in conjunction with other form controls

https://github.com/rancher/dashboard/pull/5163

**Comment by lvuch** [comment]
growl alerts https://github.com/rancher/dashboard/pull/5164

**Comment by lvuch** [comment]
icons:
https://github.com/rancher/dashboard/pull/5166

**Comment by lvuch** [comment]
growl alerts:
https://github.com/rancher/dashboard/pull/5168

**Pull request closed without merging**